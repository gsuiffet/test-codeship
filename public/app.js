var React = require('react');

class App extends React.Component {
    constructor() {
        super();
    }
    render() {
        return (
            <div>
                <h1>ma 1ere App @</h1>
            </div>
        )
    }
}

module.exports = App;