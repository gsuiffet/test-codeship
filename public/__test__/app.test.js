import React from 'react';
import renderer from 'react-test-renderer';
import {App} from '../app.js'

describe('components', function() {
    describe('<DashboardX/>', function() {
        it('renders correctly', function() {
            var component = renderer.create(App);
            const json = component.toJSON();
            expect(json).toMatchSnapshot();
        });
    });
});