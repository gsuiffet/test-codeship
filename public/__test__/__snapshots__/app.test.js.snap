// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <DashboardX/> renders correctly 1`] = `null`;
